starknet_py/tests/e2e/docs/code_examples/test_websocket_client.py

```diff
@@ -210,39 +202,39 @@ def handler(transaction_status_notification: TransactionStatusNotification):
     assert unsubscribe_result is True
 
 
-@pytest.mark.skip(
-    reason="TODO(cptartur): Unskip when adding changes for websockets in RPC 0.9.0 "
-)
+@pytest.mark.skip(reason="TODO(cptartur): Investigate why tx hashes assertion fails")
```

Comment:
It's either some weird bug in our tests/impl or a bug in devnet. The returned receipt doesn't match the hash of the tx that is being sent.